Library/Formula/libsass.rb

```diff
@@ -18,15 +18,11 @@ class Libsass < Formula
 
   def install
     ENV.cxx11
-    ENV["LIBSASS_VERSION"] = "HEAD" if build.head?
+    ENV["LIBSASS_VERSION"] = "3.2.0"
```

Comment:
Ok. Maybe it's worth switching this formula to use a Git checkout so this doesn't need to be set/updated each time?

HEAD means it was installed from your Git repository's `master` branch.